ui/apps/ui/src/app/dictionary/translateDictionaryValue.ts

```diff
@@ -11,6 +11,18 @@ import { interoperabilityGuidelinesAuthorTypeDictionary } from './interoperabili
 import { trainingQualificationsDictionary } from './trainingQualificationsDictionary';
 import { trainingDomainDictionary } from './trainingDomainDictionary';
 
+function cleanGuidelineProvider(str: string): string {
+  const index = str.indexOf('.');
+  const newValue = str.substring(index + 1).replace(/_+|-+/g, ' ');
+  return newValue;
+}
+
+function cleanValueTypeForProvider(valueType: string): string {
+  const index = valueType.indexOf('>');
```

Comment:
the assumption that data is not nested more than 2 levels in depth (just one ">" - so: category > subcategory only)
@agpul scientific domains cannot be more nested?

Reply:
it was confirmed with @agpul that is a safe assumption. I give my approval